src/static/riot/datasets/management.tag

```diff
@@ -276,14 +280,13 @@
 
         self.update_datasets = function (filters) {
             filters = filters || {}
-            let show_datasets_created_by_comp = $(self.refs.auto_created).prop('checked')
-            if (!show_datasets_created_by_comp) {
-                filters.was_created_by_competition = false
-            }
+            filters.was_created_by_competition = $(self.refs.auto_created).prop('checked')
+            filters._public = $(self.refs.show_public).prop('checked')
+            filters._type = "dataset"
             CODALAB.api.get_datasets(filters)
                 .done(function (data) {
                     self.datasets = data.results
-                    self.datasets = self.filter_out_submissions(self.datasets)
+                    //self.datasets = self.filter_out_submissions(self.datasets)
```

Comment:
If this line is not used it should be removed instead of commented

Reply:
Done